src/operator/controllers/kafkaserverconfig_controller_test.go

```diff
@@ -134,6 +135,10 @@ func (s *KafkaServerConfigReconcilerTestSuite) reconcile(namespacedName types.Na
 		res, err = s.reconciler.Reconcile(context.Background(), ctrl.Request{
 			NamespacedName: namespacedName,
 		})
+
+		if err != nil && strings.Contains(err.Error(), "the object has been modified; please apply your changes to the latest version and try again") {
```

Comment:
use `if k8serrors.IsConflict` instead - see examples elsewhere

Reply:
Did it